challenges/functions.js

```diff
@@ -6,7 +6,24 @@
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
+function consume (param1, param2, cb) 
+{
+  console.log(cb(param1, param2)); 
+} 
 
+function add(x, y) {
+  return x + y; 
+} 
+
+consume(2,2,add); 
+ 
+function multiply(x, y) {
+  return x * y; 
+} 
+
+consume (10,16,multiply); 
+
+function greeting(first_name,last_name)
```

Comment:
You're on the right track here! Everything in the functions.js file is looking good so far!